### DISCUSSION TITLE
Great job on the Smart TV block list

### DISCUSSION THREAD

**Discussion by ghost** [blocklistproject/Lists#265] [General]
I was shocked when I installed Pi-Hole and found that our Samsung Smart TV was the number one requester of ads, tracking, and other stuff by a factor of 4, that is, the Samsung TV made four times more DNS requests than any other system in our home.  The only reason we purchased the TV was to watch movies from my local Plex server and for my son to watch some YouTube videos.

Thanks for creating the Smart TV block list.

**Comment by blocklistproject** [comment]
You are very welcome. I am glad that we are able to help block unwanted requests from users devices. Please let us know if there is anything else that we can work on.

**Reply by CYB3RL4MBD4** [reply]
Many thanks for this list, I'm glad to use it too 😀

**Reply by blocklistproject** [reply]
Very welcome!
